app/views/provider_interface/offer/ske_lengths/_form.html.erb

```diff
@@ -0,0 +1,17 @@
+<%= form_with model: @wizard, url: provider_interface_application_choice_offer_ske_length_path(@application_choice), method: method do |f| %>
+  <%= f.govuk_error_summary %>
+
+  <span class="govuk-caption-l"><%= t('caption.make_offer', name: @application_choice.application_form.full_name) %></span>
+  <% if language_ske? %>
+    <% if @wizard.ske_conditions.many? %>
+      <h1 class="govuk-heading-l"><%= t('.ske_language_length_many_title') %></h1>
+      <p class="govuk-hint">One language course must be 8 weeks. The other course can be between 8 and 28 weeks.</p>
```

Comment:
have we got these numbers (8 and 28) defined in constants somewhere?

Reply:
:+1: We have an array of lengths so we can use that.